### ISSUE TITLE
Add ARIA 1.3 annotation roles

### ISSUE THREAD

**Issue by straker** [dequelabs/axe-core#3122]
[ARIA 1.3 annotations](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Annotations#aria_annotations_features) adds a few new roles. We should add the roles to `standards/aria-roles` so users can use them and not have axe-core fail them as unknown roles. Using the roles doesn't cause any problems and will only add information for browsers that support them.

```html
<div role="comment"></div>
```

**Note:** This ticket _does not include_ adding `aria-description` attribute.
